.circleci/config.yml

```diff
@@ -68,7 +68,7 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker:
-          version: 20.10.11
+          version: 27
```

Comment:
Can we just drop the version and use the default? I think this was previously overridden to enable a particular feature.